projects/addon-doc/services/theme-night.service.ts

```diff
@@ -11,17 +11,25 @@ import {
     providedIn: 'root',
 })
 export class TuiThemeNightService extends BehaviorSubject<boolean> {
-    constructor(
-        @Inject(WINDOW) readonly win: Window,
-        @Inject(LOCAL_STORAGE) readonly storage: Storage,
-        @Inject(TUI_THEME_NIGHT_STORAGE_KEY) readonly key: string,
-        @Inject(TUI_USE_DEFAULT_NIGHT_THEME) readonly useDefaultNightTheme: boolean,
-    ) {
+    readonly useDefaultNightTheme = inject(TUI_USE_DEFAULT_NIGHT_THEME);
+    readonly storage: Storage;
+    readonly win: Window;
+    readonly key: string;
+
+    constructor() {
+        const win = inject(WINDOW);
+        const storage = inject(LOCAL_STORAGE);
+        const key = inject(TUI_THEME_NIGHT_STORAGE_KEY);
+
         super(
             storage.getItem(key) === 'true' ||
                 (storage.getItem(key) === null &&
                     win.matchMedia('(prefers-color-scheme: dark)').matches),
         );
+
+        this.win = win;
+        this.storage = storage;
+        this.key = key;
```

Comment:
Probably, for this case the previous approach is better

Reply:
fixed